examples/utils/bundle-size-tests/src/odspDriver.ts

```diff
@@ -2,9 +2,11 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { OdspDocumentServiceFactory } from "@fluidframework/odsp-driver";
+import { OdspDocumentServiceFactory, prefetchLatestSnapshot } from "@fluidframework/odsp-driver";
 
 export function apisToBundle() {
      // Pass through dummy parameters, this file is only used for bundle analysis
     new OdspDocumentServiceFactory(undefined as any, undefined as any);
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
I think the goal is to measure impact of prefetchLatestSnapshot independently from OdspDocumentServiceFactory.
There might be a bunch of shared code between the two, so measuring combined bundle will not tell us the impact to the host that will use prefetchLatestSnapshot outside of iframe, and factory inside of iframe

Reply:
Jatin if this is the case you can move this call into its own file in the same location and add it to the list of entry points in the package's webpack.config.js file (module.exports.entry)

Reply:
+1 to what Vlad said

Reply:
So the total size increase seems to be 29KB which is reflected in total size. We don't have a separate entry for the api as there is no baseline build.